recipes/default.rb

```diff
@@ -15,3 +15,114 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# Instal PHP + Modules
+node.default['php']['version'] = '7.4'
+node.default['osl-php']['use_ius'] = true
+node.default['osl-php']['use_opcache'] = true
+node.default['osl-php']['php_packages'] = %w(fpm cli imap json mysqlnd mbstring)
+
+include_recipe 'osl-php'
+include_recipe 'osl-apache'
+include_recipe 'osl-apache::mod_php'
+include_recipe 'osl-mysql::client'
+
+# Following this tutorial: https://linuxize.com/post/set-up-an-email-server-with-postfixadmin/
+
+group 'vmail' do
+  gid 5000
```

Comment:
The linuxize.org article David was following says "Since we are configuring a mail server with virtual users we need one system user which will be the owner of all mailboxes and will be used by the virtual users to access their email messages on the server," which is why I'm assuming he added it and the vmail user below